e2etests/cloud/run_cloud_deployment_test.go

```diff
@@ -21,16 +21,24 @@ import (
 	"github.com/terramate-io/terramate/cmd/terramate/cli/clitest"
 	. "github.com/terramate-io/terramate/e2etests/internal/runner"
 	"github.com/terramate-io/terramate/errors"
+	"github.com/terramate-io/terramate/git"
 	"github.com/terramate-io/terramate/test"
 	"github.com/terramate-io/terramate/test/sandbox"
 )
 
-const testRemoteRepoURL = "terramate.io/terramate-io/dummy-repo.git"
+const testRemoteRepoURL = "git@github.com:terramate-io/terramate.git"
 
 var normalizedTestRemoteRepo string
 
 func init() {
-	normalizedTestRemoteRepo = cloud.NormalizeGitURI(testRemoteRepoURL)
+	r, err := git.NormalizeGitURI(testRemoteRepoURL)
```

Comment:
*nitpick* Why don't we use `TestMain` instead?

Reply:
each test file has different requirements for the "git remote URL". For example, the `preview` tests require a "github.com" for now (I reverted the change to use the same repo URL).

Reply:
I would then keep it in the test but not important